integrations/integrations.php

```diff
@@ -77,13 +73,15 @@ public function load_active(): void {
 	 * @param string $slug   A unique identifier for the integration.
 	 * @param array  $config An associative array of configuration values for the integration.
 	 *
+	 * @throws InvalidArgumentException Exception if invalid argument are passed.
+	 *
 	 * @private
 	 */
-	public function activate( string $integration_slug, array $config = [] ): void {
-		$integration = $this->get( $integration_slug );
+	public function activate( string $slug, array $config = [] ): void {
+		$integration = $this->get( $slug );
 
 		if ( null === $integration ) {
-			throw new InvalidArgumentException( sprintf( 'VIP Integration with slug "%s" is not a registered integration.', $integration_slug ) );
+			throw new InvalidArgumentException( sprintf( 'VIP Integration with slug "%s" is not a registered integration.', $integration ) );
```

Comment:
`$integration` will be `null`.

```suggestion
			throw new InvalidArgumentException( sprintf( 'VIP Integration with slug "%s" is not a registered integration.', $slug ) );
```